Source/CarthageKit/Swift3Shims.swift

```diff
@@ -1,6 +1,8 @@
 import Foundation
 
 #if swift(>=3)
```

Comment:
We still need the `Result.fanout` shim (which is not included in 3.1.0) and I wanted to minimize the diff in this pull request. Let's address it in a separate pull request.